scripts/jenkins/cloud/ansible/roles/input_model_generator/templates/mkcloud.config.j2

```diff
@@ -93,6 +93,10 @@ export hacloud=1
 export clusterconfig="{{ ns.cluster_config | join(':') }}"
 export want_sbd=1
 {% endif %}
+{% if ssl_enabled %}
+export want_all_ssl=1
```

Comment:
This is not mentioned in the commit message.

Reply:
I added this to the commit message